scripts/generate_code.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+CDIR=$(dirname $0)
+XDIR=$(realpath $CDIR/..)
```

Comment:
`realpath` is not available in the CircleCI environment.

Reply:
Gone.